packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts

```diff
@@ -160,24 +160,6 @@ export const getByViewerHandler = async ({ ctx, input }: GetByViewerOptions) =>
   });
 
   const userEventTypes = user.eventTypes.map(mapEventType);
-  // backwards compatibility, TMP:
-  const typesRaw = (
-    await prisma.eventType.findMany({
-      where: {
```

Comment:
This query is exactly the same as what's above this in the main query to get the user.